client-side/src/stories/popup/popup.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Modal from 'react-modal';
+import CloseIcon from '@mui/icons-material/Close';
+import './popup.scss';  
+
+export const popup = ({ popupContent, isOpen, onClose }) => {
+  const close = () => {
+    onClose();
+  };
+
+  return (
+    <Modal isOpen={isOpen} className="ReactModal__Content" overlayClassName="ReactModal__Overlay">
+      <button className="modal-button" onClick={close}><CloseIcon/></button>
+      <div className="modal-body">
+        {popupContent}
+      </div>
+    </Modal>
+  );
```

Comment:
add propTypes

Reply:
popupContent should have defaultValue because he didn't marked ad isRequired